### ISSUE TITLE
Do you want to try arm-2d

### ISSUE THREAD

**Issue by GorgonMeducer** [semiversus/megacard_display#1]
Hi,

I hope this message finds you well.

I haven't talked to you for quite a long time. I notice this project, and it appears that you are creating some LCD modules.
I have participated in an embedded 2d image processing project, code name arm-2d. 

https://github.com/ARM-software/EndpointAI/tree/master/Kernels/Research/Arm-2D

Are you interested in trying arm-2d in this new project?

Cheers,
Gabriel

**Comment by semiversus** [comment]
Hi Gabriel,

nice to hear from you! Hope everything is fine on your side. This repository will contain something very basic compared to the arm-2d project ;-) It's a simple text display (16x2) connected to a micro controller board used in the school I'm teacher at.

I'll have a look at the library. Two month ago I ported the NES emulator to the [STM32F746G-DISCO](https://www.st.com/en/evaluation-tools/32f746gdiscovery.html) board. I used the basic pixel based PPU implementation, which is very limiting. Without optimizations I only reached ~60% speed.

Bye,
  Günther
